packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -33,24 +34,93 @@ Requires:       iptables
 Requires:       procps
 Requires:       dhclient
 
-Provides:       bundled(docker)
+# find ../../ecs-init/vendor -name \*.go -exec dirname {} \; | sort | uniq | sed 's,^.*ecs-init/vendor/,,; s/^/bundled(golang(/;s/$/))/;' | sed 's/^/Provides:\t/' | expand -
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws))
```

Comment:
Again, for my own understanding, does this mean that every time something new is vendored for ecs-init package, something needs to be added here as well? Can you please add some documentation regarding that as well if that's the case?

Reply:
Yes this should be updated when dependencies are updated. I'll find a good place to document this requirement.